### DISCUSSION TITLE
How to add Labels/ Assign people for Pull Requests?

### DISCUSSION THREAD

**Discussion by 19valentin99** [pykale/pykale#242] [Q&A] [answered]
I would like to ask how it is possible to add: Labels or to Assign people to check my pull request because, when I was submitting the pull request I couldn't see the buttons that usually appear on the examples that are on the internet and I can not modify them afterwards.
What I see is something like this:
![image](https://user-images.githubusercontent.com/27396461/137325736-fa5e7553-b688-49fb-938f-4aeaed201d6c.png)

Thank you.

**Comment by haipinglu** [comment]
You need the right to do so. I just gave it to you.

**Reply by 19valentin99** [reply]
Thank You.

**Comment by haipinglu** [comment, marked as answer]
The template was not clear in this aspect. I will update.

**Reply by 19valentin99** [reply]
Thank You.
